src/App.vue

```diff
@@ -351,6 +362,34 @@ const { isOverDropZone } = useDropZone(dropZoneRef, {
   // whether to prevent default behavior for unhandled events
   preventDefaultForUnhandled: false,
 })
+
+// save spec to localStorage
+watchDebounced(specText, (newValue) => {
+  if (!isCleared.value && newValue && newValue.trim() !== defaultSpec.trim()) {
```

Comment:
if the user clears out the spec, wouldn't you still want to update localStorage?